_articles/building-community.md

```diff
@@ -32,6 +32,7 @@ Start with your documentation:
 
 * **Make it easy for someone to use your project.** [A friendly README](../starting-a-project/#writing-a-readme) and clear code examples will make it easier for anyone who lands on your project to get started.
 * **Clearly explain how to contribute**, using [your CONTRIBUTING file](../starting-a-project/#writing-your-contributing-guidelines) and keeping your issues up-to-date.
+* **Beginner friendly issues**, as a open source maintainer you should create few beginner friendly tickets for people who have never contributed in the open source project or say any open source project, you can label such issues as good first issues. You should encourage contributors for resolving such issues before they pick more severe issues during the journey of working with an open source project.
```

Comment:
I would rather not recommend synthetically creating beginner friendly tickets, rather labeling any that exist/get created. Also I'm not sure about the last sentence. This bullet could be one brief sentence, and link to https://help.github.com/en/articles/helping-new-contributors-find-your-project-with-labels

Reply:
I agree with Mike. See my suggested change.